script/electron-builder-linux.yml

```diff
@@ -7,5 +7,14 @@ linux:
   target:
     - deb
     - rpm
+    - snap
     - AppImage
   maintainer: 'GitHub, Inc <opensource+desktop@github.com>'
+snap:
+  confinement: 'classic'
+  stagePackages:
+    - default
+    - git-core
+    - libcurl3-gnutls
```

Comment:
A heads-up: from `1.4.0` the embedded Git package will link to `libcurl` directly - I've documented the new dependencies in https://github.com/desktop/desktop/pull/5478 for reference.

Reply:
Are you vendoring `libcurl` in GitHub Desktop 1.4.0? If you are going to start bundling your own `libcurl` with GitHub Desktop then `libcurl3-gnutls` can be dropped from `staged-packages:`.

Reply:
> Are you vendoring `libcurl` in GitHub Desktop 1.4.0?

No, shared libraries still.